src/app/components/Curation/getComponentName/index.test.ts

```diff
@@ -28,9 +30,24 @@ describe('getComponentName', () => {
     ${RANKED}       | ${NORMAL}            | ${MOST_READ}
     ${'fake-style'} | ${'fake-prominence'} | ${null}
   `(
-    'should return $expected when visual style is $visualStyle and visual prominence is $visualProminence',
+    'should return $expected when visual style is $visualStyle and visual prominence is $visualProminence and curation type is $curationType',
```

Comment:
Let's delete reference to curation type since we're no longer using it
```suggestion
    'should return $expected when visual style is $visualStyle and visual prominence is $visualProminence',
```

Reply:
done

Reply:
done